server/main/doc.go

```diff
@@ -0,0 +1,26 @@
+// Package classification Focalboard Server
```

Comment:
Moving the general documentation to an independent file fixes the home path generated reference.